eox_hooks/apps.py

```diff
@@ -4,8 +4,32 @@
 
 from __future__ import unicode_literals
 
+from importlib import import_module
+
 from django.apps import AppConfig
 
+OPENEDX_HOOKS_TRIGGER_MODULE = "openedx.core.lib.triggers.v1"
```

Comment:
A setting is not possible, is it ?

Reply:
I tried it and an error was raised:

`raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")                                                                            django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.   `

Maybe it's not possible because the settings are not available when loading the app?